integrations/sveltekit/src/custom-theme.ts

```diff
@@ -0,0 +1,125 @@
+/**
+ * The custom theme for Next.js
+ */
+export const customTheme = `
```

Comment:
we need @cam here I think

Reply:
Yeah I just lifted this straight from Next.js, could use some guidance